src/main/java/com/example/home_work_4/StringTotal.java

```diff
@@ -0,0 +1,393 @@
+package com.example.home_work_4;
+
+import java.util.Scanner;
+
+public class StringTotal {
+
+    public static void main(String[] args) {
+        //String1_1
+        String s1 = "<<>>";
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter your word: ");
+        String yourWord = scanner.nextLine();
```

Comment:
You don't need to use `scanner` at all. Please remove it completely from this class